Straight_skeleton_2/examples/Straight_skeleton_2/exterior_offset_of_multiple_polygons_with_holes.cpp

```diff
@@ -0,0 +1,131 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+
+#include <CGAL/Polygon_with_holes_2.h>
+#include <CGAL/create_offset_polygons_from_polygon_with_holes_2.h>
+
+#include <boost/shared_ptr.hpp>
+
+#include <CGAL/draw_polygon_with_holes_2.h>
+
+
+#include <vector>
+#include <cassert>
+
+typedef CGAL::Exact_predicates_inexact_constructions_kernel K ;
+
+typedef K::Point_2                    Point ;
+typedef CGAL::Polygon_2<K>            Polygon ;
```

Comment:
Elsewhere we have `typedef CGAL::Polygon_2<K>  Polygon_2 ;`  and I think this is because the token `Polygon`  is already used by some Windows header file,.  This is probalbly the reason for the failing VC testsuite of today.